### PR TITLE
raidboss: add Shaaloani A-rank hunt marks

### PR DESCRIPTION
**Pull request by wexxlee**
Keheni's forced march mech is another one where a countdown would be pretty useful.  Left a note in case #358 or something similar gets added.

### CODE REVIEW

File: ui/raidboss/data/07-dt/hunts/shaaloani.ts [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+type ForcedMarch = 'forward' | 'backward' | 'left' | 'right';
+const effectIdToForcedMarchDir: { [id: string]: ForcedMarch } = {
+  871: 'forward', // Forward March
+  872: 'backward', // About Face
+  873: 'left', // Left Face
+  874: 'right', // Right Face
+};
+
+// For Yehehe's directional debuffs and attacks, use 0-3 for relative direction
+// e.g. 0=front, 1=right, 2=back,3=left
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/hunts/shaaloani.ts:19]:
```suggestion
// e.g. 0=front, 1=right, 2=back, 3=left
```
whitespace nit-pick

File: ui/raidboss/data/07-dt/hunts/shaaloani.ts [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        backward: {
+          en: 'Forced March: Backward (later)',
+        },
+        left: {
+          en: 'Forced March: Left (later)',
+        },
+        right: {
+          en: 'Forced March: Right (later)',
+        },
+      },
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/hunts/shaaloani.ts:96]: unrelated to review: Forced March is a somewhat common mechanic, I wonder if we should add it to `outputs.ts`

**Review by xiashtra — APPROVED** [review]
minor comments

File: ui/raidboss/data/07-dt/hunts/shaaloani.ts (hunk 2 above)
Comment (wexxlee) [reply ui/raidboss/data/07-dt/hunts/shaaloani.ts:96]: > unrelated to review: Forced March is a somewhat common mechanic, I wonder if we should add it to `outputs.ts`

Not opposed at all.  But it seems that usually Forced March is combined with another mech (like chariot/dynamo), so in most cases, I think we'd want the trigger output for the mech to have a combined call, whereas a call specific to just Forced March would likely be something like this (flagging that is has been applied for later, plus maybe a countdown).

**Pull request merged**